launch-script/src/main.rs

```diff
@@ -293,6 +293,52 @@ fn main() {
                         .help("Proposal for vote")
                 )
             )
+            .subcommand(SubCommand::with_name("create-start-evm")
+                .about("Create proposal for Neon EVM start")
+                .arg(
+                    Arg::with_name("collateral-pool-base")
+                        .long("collateral-pool-base")
+                        .required(true)
+                        .takes_value(true)
+                        .value_name("COLLATERAL_POOL_BASE")
+                        .help("Collateral pool base")
+                )
+                .arg(
+                    Arg::with_name("evm-loader")
+                        .long("evm-loader")
+                        .required(true)
+                        .takes_value(true)
+                        .value_name("EVM_LOADER_ADDRESS")
+                        .help("EVM loader address")
+                )
+            )
+            .subcommand(SubCommand::with_name("create-evm-from-buffer")
```

Comment:
Maybe rename this command to `create-upgrade-evm`?

Reply:
Fixed